packages/smarthr-ui/src/components/Experimental/SideMenu/SideMenu.tsx

```diff
@@ -7,7 +7,10 @@ const sideMenu = tv({
   base: 'smarthr-ui-SideMenu shr-list-none shr-py-0.5',
 })
 
-type Props = Pick<ComponentPropsWithoutRef<typeof Base>, 'radius' | 'layer' | 'className'> & {
+type Props = Pick<
```

Comment:
[imo]
@TakahiroHimi 近頃SmartHR UIの中だと以下のような記述のほうが増えてきている印象があります〜！
```suggestion
type Props = PropsWithChildren<Pick<....
```

Reply:
@atzzCokeK 
こっちの方がスマートに思えたので変更しました！ありがとうございます！！！
https://github.com/kufu/smarthr-ui/pull/4919/commits/a698ca8116366abe47c46b341dfe1456f677b0de